### DISCUSSION TITLE
Changing screens causes an AMD hang on relaunch.

### DISCUSSION THREAD

**Discussion by SoffiMeower** [ppy/osu#21254] [Q&A]
### Type

Crash to desktop

### Bug description

So, osu!lazer suddenly opened on my 2nd monitor/TV, I changed it to my gaming display, all was fine for that run. However, attempting to relaunch now always has the AMD driver crash. Attempting to only have 1 monitor connected doesn't work.
Win+Ctrl+Shift+B also didn't work (hard driver reset)

### Screenshots or videos

![image](https://user-images.githubusercontent.com/45863583/202073666-c6391e06-797e-441e-92ed-dc9b9f43267b.png)
[AMD Bug Reporting Form Submitted Data 20221115T220616.zip](https://github.com/ppy/osu/files/10017783/AMD.Bug.Reporting.Form.Submitted.Data.20221115T220616.zip)
![image](https://user-images.githubusercontent.com/45863583/202074223-24c96ac3-33e5-4ec6-81e5-24d25393f1f4.png)

### Version

2022.1101.0-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/10017809/runtime.log)
[performance.log](https://github.com/ppy/osu/files/10017810/performance.log)
[updater.log](https://github.com/ppy/osu/files/10017811/updater.log)
[database.log](https://github.com/ppy/osu/files/10017812/database.log)
[input.log](https://github.com/ppy/osu/files/10017813/input.log)

**Comment by peppy** [comment]
Will probably have to wait for either AMD to fix this, or for us to support Direct X / vulkan so you can avoid whatever is causing the crash in the first place.
